### PR TITLE
add linting and minify process

### PR DESCRIPTION
**Pull request by adslaton**
# Why

Add linting and minify process.
# How

Files in /dist will now be dynamically created upon running `npm run build`. Development should now occur in /src.
# Testing
1. run npm install
2. to see linting tests run: `npm test`
3. to see the build created run: `npm run build`

### CODE REVIEW

File: dist/injector.js [hunk 1]
```diff
@@ -24,7 +24,6 @@ window.FAI.bundleHost = window.FAI.bundleHost || '/';
         featureExecuteSuccess,
         featureLoadSuccess,
         scanForFeature,
-        loadFeature,
```
Comment (martytheparty) [inline dist/injector.js:27]: What does this do?

Comment (adslaton) [reply dist/injector.js:27]: Nothing.... it was unused so I removed it.

Comment (martytheparty) [reply dist/injector.js:27]: Cool.  thanks

**Review by martytheparty — APPROVED** [review]
Great job!

**Pull request merged**